forta-uniswap-event-py/README.md

```diff
@@ -0,0 +1,27 @@
+# Large Uniswap Detector Agent
+
+## Description
+
+This agent detects large uniswap transactions on ETH liquidity pools
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- AE-UNISWAP-LARGESWAP-EVENT
```

Comment:
With all of the other handlers, we have converged on the following format for README.md entries:
```
- AE-<PROTOCOL>-<BRIEF-ALERT-NAME>
  - Fired when <situation when alert is fired>
  - Severity is always set to "<severity level>"
  - Type is always set to "<type>"
  - Metadata field contains <additional alert data that metadata field contains>

  <Any additional information about handler, such as required API keys>
```